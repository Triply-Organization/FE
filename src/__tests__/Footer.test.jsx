import '@testing-library/jest-dom/extend-expect';
import { fireEvent, render } from '@testing-library/react';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { describe, expect, test } from 'vitest';

import Footer from '../components/Footer/Footer';

describe('component/Footer', () => {
  test('render correctly', () => {
    expect(
      render(
        <Router>
          <Footer />
        </Router>,
      ),
    ).toMatchSnapshot();
  });

  test('should correctly finish', () => {
    const ctn = render(
      <Router>
        <Footer />
      </Router>,
    );

    const form = ctn.queryByTestId('footer_form');
    const input = ctn.queryByTestId('footer-input');

    // console.log(input);
    expect(form).toBeInTheDocument();
    fireEvent.change(input, { target: { value: 'gg@gmail.com' } });
    expect(input.value).toBe('gg@gmail.com');
    fireEvent.submit(form);

    // fireEvent.submit(submit);
  });
});
