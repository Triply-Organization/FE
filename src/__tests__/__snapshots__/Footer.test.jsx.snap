// Vitest Snapshot v1

exports[`component/Footer > render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer__wrapper"
      >
        <div
          class="section-4"
        >
          <div
            class="section-4__contact-wrapper"
          >
            <p
              class="section-4__text"
            >
              home.banner.telephone
            </p>
            <a
              class="section-4__link"
              href="#"
            >
              <span
                class="section-4__link-icon"
              >
                <svg
                  enable-background="new 0 0 48 48"
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  version="1"
                  viewBox="0 0 48 48"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M6,36V8c0-2.2,1.8-4,4-4h14c2.2,0,4,1.8,4,4v28c0,2.2-1.8,4-4,4H10C7.8,40,6,38.2,6,36z"
                    fill="#37474F"
                  />
                  <path
                    d="M24,7H10C9.4,7,9,7.4,9,8v25c0,0.6,0.4,1,1,1h14c0.6,0,1-0.4,1-1V8C25,7.4,24.6,7,24,7z"
                    fill="#BBDEFB"
                  />
                  <rect
                    fill="#78909C"
                    height="2"
                    width="6"
                    x="14"
                    y="36"
                  />
                  <path
                    d="M20,40V12c0-2.2,1.8-4,4-4h14c2.2,0,4,1.8,4,4v28c0,2.2-1.8,4-4,4H24C21.8,44,20,42.2,20,40z"
                    fill="#E38939"
                  />
                  <path
                    d="M38,11H24c-0.6,0-1,0.4-1,1v25c0,0.6,0.4,1,1,1h14c0.6,0,1-0.4,1-1V12C39,11.4,38.6,11,38,11z"
                    fill="#FFF3E0"
                  />
                  <circle
                    cx="31"
                    cy="41"
                    fill="#A6642A"
                    r="1.5"
                  />
                </svg>
              </span>
               
              <span
                class="section-4__link-words"
              >
                +(84) 1800 - 333 5555
              </span>
            </a>
          </div>
          <div
            class="section-4__contact-wrapper"
          >
            <p
              style="opacity: 0;"
            >
              a
            </p>
            <a
              class="section-4__link"
              href="#"
            >
              <span
                class="section-4__link-icon"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 24 24"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                  <path
                    d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10h5v-2h-5c-4.34 0-8-3.66-8-8s3.66-8 8-8 8 3.66 8 8v1.43c0 .79-.71 1.57-1.5 1.57s-1.5-.78-1.5-1.57V12c0-2.76-2.24-5-5-5s-5 2.24-5 5 2.24 5 5 5c1.38 0 2.64-.56 3.54-1.47.65.89 1.77 1.47 2.96 1.47 1.97 0 3.5-1.6 3.5-3.57V12c0-5.52-4.48-10-10-10zm0 13c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                  />
                </svg>
              </span>
               
              <span
                class="section-4__link-words"
              >
                support@triply.com
              </span>
            </a>
          </div>
          <div
            class="section-4__follow-us"
          >
            <p
              class="section-4__text"
            >
              home.banner.follow
            </p>
            <div
              class="section-4__wrapper-button"
            >
              <button
                class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
                type="button"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"
                  />
                </svg>
              </button>
              <button
                class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
                type="button"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
                  />
                </svg>
              </button>
              <button
                class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
                type="button"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8.051 1.999h.089c.822.003 4.987.033 6.11.335a2.01 2.01 0 0 1 1.415 1.42c.101.38.172.883.22 1.402l.01.104.022.26.008.104c.065.914.073 1.77.074 1.957v.075c-.001.194-.01 1.108-.082 2.06l-.008.105-.009.104c-.05.572-.124 1.14-.235 1.558a2.007 2.007 0 0 1-1.415 1.42c-1.16.312-5.569.334-6.18.335h-.142c-.309 0-1.587-.006-2.927-.052l-.17-.006-.087-.004-.171-.007-.171-.007c-1.11-.049-2.167-.128-2.654-.26a2.007 2.007 0 0 1-1.415-1.419c-.111-.417-.185-.986-.235-1.558L.09 9.82l-.008-.104A31.4 31.4 0 0 1 0 7.68v-.123c.002-.215.01-.958.064-1.778l.007-.103.003-.052.008-.104.022-.26.01-.104c.048-.519.119-1.023.22-1.402a2.007 2.007 0 0 1 1.415-1.42c.487-.13 1.544-.21 2.654-.26l.17-.007.172-.006.086-.003.171-.007A99.788 99.788 0 0 1 7.858 2h.193zM6.4 5.209v4.818l4.157-2.408L6.4 5.209z"
                  />
                </svg>
              </button>
              <button
                class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
                type="button"
              >
                <svg
                  fill="currentColor"
                  height="1em"
                  stroke="currentColor"
                  stroke-width="0"
                  viewBox="0 0 16 16"
                  width="1em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M8 0C5.829 0 5.556.01 4.703.048 3.85.088 3.269.222 2.76.42a3.917 3.917 0 0 0-1.417.923A3.927 3.927 0 0 0 .42 2.76C.222 3.268.087 3.85.048 4.7.01 5.555 0 5.827 0 8.001c0 2.172.01 2.444.048 3.297.04.852.174 1.433.372 1.942.205.526.478.972.923 1.417.444.445.89.719 1.416.923.51.198 1.09.333 1.942.372C5.555 15.99 5.827 16 8 16s2.444-.01 3.298-.048c.851-.04 1.434-.174 1.943-.372a3.916 3.916 0 0 0 1.416-.923c.445-.445.718-.891.923-1.417.197-.509.332-1.09.372-1.942C15.99 10.445 16 10.173 16 8s-.01-2.445-.048-3.299c-.04-.851-.175-1.433-.372-1.941a3.926 3.926 0 0 0-.923-1.417A3.911 3.911 0 0 0 13.24.42c-.51-.198-1.092-.333-1.943-.372C10.443.01 10.172 0 7.998 0h.003zm-.717 1.442h.718c2.136 0 2.389.007 3.232.046.78.035 1.204.166 1.486.275.373.145.64.319.92.599.28.28.453.546.598.92.11.281.24.705.275 1.485.039.843.047 1.096.047 3.231s-.008 2.389-.047 3.232c-.035.78-.166 1.203-.275 1.485a2.47 2.47 0 0 1-.599.919c-.28.28-.546.453-.92.598-.28.11-.704.24-1.485.276-.843.038-1.096.047-3.232.047s-2.39-.009-3.233-.047c-.78-.036-1.203-.166-1.485-.276a2.478 2.478 0 0 1-.92-.598 2.48 2.48 0 0 1-.6-.92c-.109-.281-.24-.705-.275-1.485-.038-.843-.046-1.096-.046-3.233 0-2.136.008-2.388.046-3.231.036-.78.166-1.204.276-1.486.145-.373.319-.64.599-.92.28-.28.546-.453.92-.598.282-.11.705-.24 1.485-.276.738-.034 1.024-.044 2.515-.045v.002zm4.988 1.328a.96.96 0 1 0 0 1.92.96.96 0 0 0 0-1.92zm-4.27 1.122a4.109 4.109 0 1 0 0 8.217 4.109 4.109 0 0 0 0-8.217zm0 1.441a2.667 2.667 0 1 1 0 5.334 2.667 2.667 0 0 1 0-5.334z"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="footer__upper"
        >
          <div
            class="footer__info-wrapper"
          >
            <div
              class="footer__heading footer__heading--center"
            >
              <img
                alt=""
                class="footer__info-logo"
                src="/src/assets/images/logo-white.png"
              />
            </div>
            <div
              class="footer__info-description"
            >
              <span
                class="footer__info-description-words"
              >
                footer.title
              </span>
            </div>
            <div
              class="footer__direct"
            >
              <div
                class="footer__info-address"
              >
                <span
                  class="footer__info-address-words"
                >
                  754 West Gartner Street Encino, CA 91316
                </span>
              </div>
              <button
                class="ant-btn ant-btn-default ant-btn-lg footer__info-view-map-btn"
                type="button"
              >
                <span>
                  cta.view
                </span>
              </button>
            </div>
          </div>
          <div
            class="footer__list-wrapper footer__list-mobile--left"
          >
            <h4
              class="ant-typography footer__heading"
            >
              footer.service.title
            </h4>
            <ul
              class="footer__list"
            >
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.service.item.booking
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.service.item.rental_car
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.service.item.hostel_world
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.service.item.triago
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.service.item.trip_advisor
                </a>
              </li>
            </ul>
          </div>
          <div
            class="footer__list-wrapper footer__list-mobile--right"
          >
            <h4
              class="ant-typography footer__heading"
            >
              footer.explore.title
            </h4>
            <ul
              class="footer__list"
            >
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.explore.item.madrid
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.explore.item.stockhome
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.explore.item.rome
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.explore.item.shanghai
                </a>
              </li>
              <li
                class="footer__item"
              >
                <a
                  class="footer__link"
                  href="#"
                >
                  footer.explore.item.tokyo
                </a>
              </li>
            </ul>
          </div>
          <div
            class="footer__detail-wrapper"
          >
            <h4
              class="ant-typography footer__heading"
            >
              footer.get_update.title
            </h4>
            <p
              class="footer__detail-words"
            >
              footer.get_update.content
            </p>
            <div
              class="footer__detail-form"
            >
              <form
                class="ant-form ant-form-horizontal"
              >
                <div
                  class="ant-row ant-form-item"
                >
                  <div
                    class="ant-col ant-form-item-control"
                  >
                    <div
                      class="ant-form-item-control-input"
                    >
                      <div
                        class="ant-form-item-control-input-content"
                      >
                        <input
                          class="ant-input ant-input-lg footer__detail-input"
                          id="followByEmail"
                          placeholder="cta.place_holder_input-email"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
                <button
                  class="ant-btn ant-btn-primary ant-btn-lg footer__detail-btn"
                  type="submit"
                >
                  <span>
                    cta.subscribe
                  </span>
                </button>
              </form>
            </div>
          </div>
        </div>
        <div
          class="footer__lower"
        >
          <div
            class="footer__lower-words"
          >
            Copyright © 2022
             
            <a
              class="footer__lower-link"
              href="#"
            >
              Triply
            </a>
            . All Rights Reserved.
          </div>
          <ul
            class="footer__lower-list"
          >
            <li
              class="footer__lower-item"
            >
              <a
                class="footer__low-link"
                href="#"
              >
                footer.contact.privacy
              </a>
            </li>
            <li
              class="footer__lower-item"
            >
              <a
                class="footer__low-link"
                href="#"
              >
                footer.contact.policy
              </a>
            </li>
            <li
              class="footer__lower-item"
            >
              <a
                class="footer__low-link"
                href="#"
              >
                footer.contact.about_us
              </a>
            </li>
            <li
              class="footer__lower-item"
            >
              <a
                class="footer__low-link"
                href="#"
              >
                footer.contact.faq
              </a>
            </li>
            <li
              class="footer__lower-item"
            >
              <a
                class="footer__low-link"
                href="#"
              >
                footer.contact.blog
              </a>
            </li>
          </ul>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer__wrapper"
    >
      <div
        class="section-4"
      >
        <div
          class="section-4__contact-wrapper"
        >
          <p
            class="section-4__text"
          >
            home.banner.telephone
          </p>
          <a
            class="section-4__link"
            href="#"
          >
            <span
              class="section-4__link-icon"
            >
              <svg
                enable-background="new 0 0 48 48"
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                version="1"
                viewBox="0 0 48 48"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M6,36V8c0-2.2,1.8-4,4-4h14c2.2,0,4,1.8,4,4v28c0,2.2-1.8,4-4,4H10C7.8,40,6,38.2,6,36z"
                  fill="#37474F"
                />
                <path
                  d="M24,7H10C9.4,7,9,7.4,9,8v25c0,0.6,0.4,1,1,1h14c0.6,0,1-0.4,1-1V8C25,7.4,24.6,7,24,7z"
                  fill="#BBDEFB"
                />
                <rect
                  fill="#78909C"
                  height="2"
                  width="6"
                  x="14"
                  y="36"
                />
                <path
                  d="M20,40V12c0-2.2,1.8-4,4-4h14c2.2,0,4,1.8,4,4v28c0,2.2-1.8,4-4,4H24C21.8,44,20,42.2,20,40z"
                  fill="#E38939"
                />
                <path
                  d="M38,11H24c-0.6,0-1,0.4-1,1v25c0,0.6,0.4,1,1,1h14c0.6,0,1-0.4,1-1V12C39,11.4,38.6,11,38,11z"
                  fill="#FFF3E0"
                />
                <circle
                  cx="31"
                  cy="41"
                  fill="#A6642A"
                  r="1.5"
                />
              </svg>
            </span>
             
            <span
              class="section-4__link-words"
            >
              +(84) 1800 - 333 5555
            </span>
          </a>
        </div>
        <div
          class="section-4__contact-wrapper"
        >
          <p
            style="opacity: 0;"
          >
            a
          </p>
          <a
            class="section-4__link"
            href="#"
          >
            <span
              class="section-4__link-icon"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10h5v-2h-5c-4.34 0-8-3.66-8-8s3.66-8 8-8 8 3.66 8 8v1.43c0 .79-.71 1.57-1.5 1.57s-1.5-.78-1.5-1.57V12c0-2.76-2.24-5-5-5s-5 2.24-5 5 2.24 5 5 5c1.38 0 2.64-.56 3.54-1.47.65.89 1.77 1.47 2.96 1.47 1.97 0 3.5-1.6 3.5-3.57V12c0-5.52-4.48-10-10-10zm0 13c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3z"
                />
              </svg>
            </span>
             
            <span
              class="section-4__link-words"
            >
              support@triply.com
            </span>
          </a>
        </div>
        <div
          class="section-4__follow-us"
        >
          <p
            class="section-4__text"
          >
            home.banner.follow
          </p>
          <div
            class="section-4__wrapper-button"
          >
            <button
              class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
              type="button"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M16 8.049c0-4.446-3.582-8.05-8-8.05C3.58 0-.002 3.603-.002 8.05c0 4.017 2.926 7.347 6.75 7.951v-5.625h-2.03V8.05H6.75V6.275c0-2.017 1.195-3.131 3.022-3.131.876 0 1.791.157 1.791.157v1.98h-1.009c-.993 0-1.303.621-1.303 1.258v1.51h2.218l-.354 2.326H9.25V16c3.824-.604 6.75-3.934 6.75-7.951z"
                />
              </svg>
            </button>
            <button
              class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
              type="button"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5.026 15c6.038 0 9.341-5.003 9.341-9.334 0-.14 0-.282-.006-.422A6.685 6.685 0 0 0 16 3.542a6.658 6.658 0 0 1-1.889.518 3.301 3.301 0 0 0 1.447-1.817 6.533 6.533 0 0 1-2.087.793A3.286 3.286 0 0 0 7.875 6.03a9.325 9.325 0 0 1-6.767-3.429 3.289 3.289 0 0 0 1.018 4.382A3.323 3.323 0 0 1 .64 6.575v.045a3.288 3.288 0 0 0 2.632 3.218 3.203 3.203 0 0 1-.865.115 3.23 3.23 0 0 1-.614-.057 3.283 3.283 0 0 0 3.067 2.277A6.588 6.588 0 0 1 .78 13.58a6.32 6.32 0 0 1-.78-.045A9.344 9.344 0 0 0 5.026 15z"
                />
              </svg>
            </button>
            <button
              class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
              type="button"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8.051 1.999h.089c.822.003 4.987.033 6.11.335a2.01 2.01 0 0 1 1.415 1.42c.101.38.172.883.22 1.402l.01.104.022.26.008.104c.065.914.073 1.77.074 1.957v.075c-.001.194-.01 1.108-.082 2.06l-.008.105-.009.104c-.05.572-.124 1.14-.235 1.558a2.007 2.007 0 0 1-1.415 1.42c-1.16.312-5.569.334-6.18.335h-.142c-.309 0-1.587-.006-2.927-.052l-.17-.006-.087-.004-.171-.007-.171-.007c-1.11-.049-2.167-.128-2.654-.26a2.007 2.007 0 0 1-1.415-1.419c-.111-.417-.185-.986-.235-1.558L.09 9.82l-.008-.104A31.4 31.4 0 0 1 0 7.68v-.123c.002-.215.01-.958.064-1.778l.007-.103.003-.052.008-.104.022-.26.01-.104c.048-.519.119-1.023.22-1.402a2.007 2.007 0 0 1 1.415-1.42c.487-.13 1.544-.21 2.654-.26l.17-.007.172-.006.086-.003.171-.007A99.788 99.788 0 0 1 7.858 2h.193zM6.4 5.209v4.818l4.157-2.408L6.4 5.209z"
                />
              </svg>
            </button>
            <button
              class="ant-btn ant-btn-circle ant-btn-default ant-btn-lg ant-btn-icon-only section-4__button"
              type="button"
            >
              <svg
                fill="currentColor"
                height="1em"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M8 0C5.829 0 5.556.01 4.703.048 3.85.088 3.269.222 2.76.42a3.917 3.917 0 0 0-1.417.923A3.927 3.927 0 0 0 .42 2.76C.222 3.268.087 3.85.048 4.7.01 5.555 0 5.827 0 8.001c0 2.172.01 2.444.048 3.297.04.852.174 1.433.372 1.942.205.526.478.972.923 1.417.444.445.89.719 1.416.923.51.198 1.09.333 1.942.372C5.555 15.99 5.827 16 8 16s2.444-.01 3.298-.048c.851-.04 1.434-.174 1.943-.372a3.916 3.916 0 0 0 1.416-.923c.445-.445.718-.891.923-1.417.197-.509.332-1.09.372-1.942C15.99 10.445 16 10.173 16 8s-.01-2.445-.048-3.299c-.04-.851-.175-1.433-.372-1.941a3.926 3.926 0 0 0-.923-1.417A3.911 3.911 0 0 0 13.24.42c-.51-.198-1.092-.333-1.943-.372C10.443.01 10.172 0 7.998 0h.003zm-.717 1.442h.718c2.136 0 2.389.007 3.232.046.78.035 1.204.166 1.486.275.373.145.64.319.92.599.28.28.453.546.598.92.11.281.24.705.275 1.485.039.843.047 1.096.047 3.231s-.008 2.389-.047 3.232c-.035.78-.166 1.203-.275 1.485a2.47 2.47 0 0 1-.599.919c-.28.28-.546.453-.92.598-.28.11-.704.24-1.485.276-.843.038-1.096.047-3.232.047s-2.39-.009-3.233-.047c-.78-.036-1.203-.166-1.485-.276a2.478 2.478 0 0 1-.92-.598 2.48 2.48 0 0 1-.6-.92c-.109-.281-.24-.705-.275-1.485-.038-.843-.046-1.096-.046-3.233 0-2.136.008-2.388.046-3.231.036-.78.166-1.204.276-1.486.145-.373.319-.64.599-.92.28-.28.546-.453.92-.598.282-.11.705-.24 1.485-.276.738-.034 1.024-.044 2.515-.045v.002zm4.988 1.328a.96.96 0 1 0 0 1.92.96.96 0 0 0 0-1.92zm-4.27 1.122a4.109 4.109 0 1 0 0 8.217 4.109 4.109 0 0 0 0-8.217zm0 1.441a2.667 2.667 0 1 1 0 5.334 2.667 2.667 0 0 1 0-5.334z"
                />
              </svg>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer__upper"
      >
        <div
          class="footer__info-wrapper"
        >
          <div
            class="footer__heading footer__heading--center"
          >
            <img
              alt=""
              class="footer__info-logo"
              src="/src/assets/images/logo-white.png"
            />
          </div>
          <div
            class="footer__info-description"
          >
            <span
              class="footer__info-description-words"
            >
              footer.title
            </span>
          </div>
          <div
            class="footer__direct"
          >
            <div
              class="footer__info-address"
            >
              <span
                class="footer__info-address-words"
              >
                754 West Gartner Street Encino, CA 91316
              </span>
            </div>
            <button
              class="ant-btn ant-btn-default ant-btn-lg footer__info-view-map-btn"
              type="button"
            >
              <span>
                cta.view
              </span>
            </button>
          </div>
        </div>
        <div
          class="footer__list-wrapper footer__list-mobile--left"
        >
          <h4
            class="ant-typography footer__heading"
          >
            footer.service.title
          </h4>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.service.item.booking
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.service.item.rental_car
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.service.item.hostel_world
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.service.item.triago
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.service.item.trip_advisor
              </a>
            </li>
          </ul>
        </div>
        <div
          class="footer__list-wrapper footer__list-mobile--right"
        >
          <h4
            class="ant-typography footer__heading"
          >
            footer.explore.title
          </h4>
          <ul
            class="footer__list"
          >
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.explore.item.madrid
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.explore.item.stockhome
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.explore.item.rome
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.explore.item.shanghai
              </a>
            </li>
            <li
              class="footer__item"
            >
              <a
                class="footer__link"
                href="#"
              >
                footer.explore.item.tokyo
              </a>
            </li>
          </ul>
        </div>
        <div
          class="footer__detail-wrapper"
        >
          <h4
            class="ant-typography footer__heading"
          >
            footer.get_update.title
          </h4>
          <p
            class="footer__detail-words"
          >
            footer.get_update.content
          </p>
          <div
            class="footer__detail-form"
          >
            <form
              class="ant-form ant-form-horizontal"
            >
              <div
                class="ant-row ant-form-item"
              >
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <input
                        class="ant-input ant-input-lg footer__detail-input"
                        id="followByEmail"
                        placeholder="cta.place_holder_input-email"
                        type="text"
                        value=""
                      />
                    </div>
                  </div>
                </div>
              </div>
              <button
                class="ant-btn ant-btn-primary ant-btn-lg footer__detail-btn"
                type="submit"
              >
                <span>
                  cta.subscribe
                </span>
              </button>
            </form>
          </div>
        </div>
      </div>
      <div
        class="footer__lower"
      >
        <div
          class="footer__lower-words"
        >
          Copyright © 2022
           
          <a
            class="footer__lower-link"
            href="#"
          >
            Triply
          </a>
          . All Rights Reserved.
        </div>
        <ul
          class="footer__lower-list"
        >
          <li
            class="footer__lower-item"
          >
            <a
              class="footer__low-link"
              href="#"
            >
              footer.contact.privacy
            </a>
          </li>
          <li
            class="footer__lower-item"
          >
            <a
              class="footer__low-link"
              href="#"
            >
              footer.contact.policy
            </a>
          </li>
          <li
            class="footer__lower-item"
          >
            <a
              class="footer__low-link"
              href="#"
            >
              footer.contact.about_us
            </a>
          </li>
          <li
            class="footer__lower-item"
          >
            <a
              class="footer__low-link"
              href="#"
            >
              footer.contact.faq
            </a>
          </li>
          <li
            class="footer__lower-item"
          >
            <a
              class="footer__low-link"
              href="#"
            >
              footer.contact.blog
            </a>
          </li>
        </ul>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
